### PR TITLE
W-10890554-db-1.13-release-kt

### PR DESCRIPTION
**Pull request by kevintroller**
Documentation for Database Connector 1.3 release.

### CODE REVIEW

File: db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+
+.Oracle Stored procedure with UDTs Flow
+image::database-oracle-udt-flow.png[The Oracle UDTs flow in Studio]
+
+
+== Using prepareStruct and prepareArray Functions
+
+The `createStruct` and `createArray` functions are deprecated and replaced by the `prepareStruct` and `prepareArray` functions.
+
+Previous Mule versions provided only the `createStruct` and `createArray` functions that created JDBC values right away. However, JDBC values must be created with the same connection used in the Database Connector operation. The `prepareStruct` and `prepareArray` functions enables to defer the JDBC values creation until the right SQL connection is available.
```
Comment (sathya0) [inline db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc]:
```suggestion
Previous Mule versions provided only the `createStruct` and `createArray` functions that created JDBC values right away. However, JDBC values must be created with the same connection used in the Database Connector operation. The `prepareStruct` and `prepareArray` functions enable you to defer the JDBC values creation until the right SQL connection is available.
```

File: db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+
+[%header,%cols]
+|===
+| Parameter | Type | Description
+| `typeName` | String | Name of the Array type to create.
+| `values` | List<Object> | Values to pass to the database.
+| `java.util.List` | List<Object> | The return value is an immutable list that holds all the information for creating an array later.
+|===
+
+`Db::prepareStruct(typeName, properties): java.util.List` function prepares an struct for sending to the database as an java.sql.Struct.
```
Comment (sathya0) [inline db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc]:
```suggestion
`Db::prepareStruct(typeName, properties): java.util.List` function prepares a struct for sending to the database as an java.sql.Struct.
```

File: db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+
+[%header,%cols]
+|===
+| Parameter | Type | Description
+| `typeName` | String | Name of the Struct type to create.
+| `properties` | List<Object> | Properties to pass to the database.
+| `java.util.List` | List<Object> | The return value is an immutable list that holds all the information for creating an struct later.
```
Comment (sathya0) [inline db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc]:
```suggestion
| `java.util.List` | List<Object> | The return value is an immutable list that holds all the information for creating a struct later.
```

**Review by sathya0 — DISMISSED** [review]

File: db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc [hunk 4]
```diff
@@ -203,14 +242,14 @@ Db::prepareArray("PEOPLE", payload map (item, index) -> ( toPerson(item) ) )
 .Transform Message configuration
 image::database-oracleudt-example-6.png[Transform Message configuration]
 
-The `prepareStruct` instructs Mule to create JDBC struct objects based on the `Name` type and their corresponding properties when a database operation is performed. In the example, types `PERSON` and `PHONE_NUMBER` are struct objects. The function expects the following parameters in order:
+When the Mule app performs a Database Connector operation, the `prepareStruct` instructs Mule to create JDBC struct objects based on the `Name` type and their corresponding properties. In the example, types `PERSON` and `PHONE_NUMBER` are struct objects. The function expects the following parameters in order:
```
Comment (niqueco) [inline db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc]: This doesn't happen when the Mule app performs an operation, is a DW function used to "prepare" data that will be sent to the db at a later time. The original text tried to account for this difference.

**Review by sathya0 — DISMISSED** [review]

File: db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc [hunk 5]
```diff
@@ -0,0 +1,381 @@
...
+
+[%header,%cols]
+|===
+| Parameter | Type | Description
+| `typeName` | String | Name of the Array type to create.
+| `values` | List<Object> | Values to pass to the database.
+| `java.util.List` | List<Object> | The return value is an immutable list that holds all the information for creating an array later.
+|===
+
+`Db::prepareStruct(typeName, properties): java.util.List` function prepares an struct for sending to the database as a java.sql.Struct.
```
Comment (sathya0) [inline db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc]:
```suggestion
`Db::prepareStruct(typeName, properties): java.util.List` function prepares a struct for sending to the database as a java.sql.Struct.
```

File: db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc [hunk 6]
```diff
@@ -0,0 +1,381 @@
...
+[%header,%cols]
+|===
+| Parameter | Type | Description
+| `typeName` | String | Name of the Struct type to create.
+| `properties` | List<Object> | Properties to pass to the database.
+| `java.util.List` | List<Object> | The return value is an immutable list that holds all the information for creating a struct later.
+|===
+
+[NOTE]
+If you have existing Mule apps that use the deprecated functions, replace  `createStruct` with `prepareStruct` and `createArray` with `prepareArray`. Additionally, remove the first parameter `configName` from the function structure.
```
Comment (sathya0) [inline db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc]:
```suggestion
If you have existing Mule apps that use the deprecated functions, replace `createStruct` with `prepareStruct` and `createArray` with `prepareArray`. Additionally, remove the first parameter `configName` from the function structure.
```

**Review by sathya0 — DISMISSED** [review]

**Review by sathya0 — APPROVED** [review]

**Pull request merged**